source/includes/_payment.md

```diff
@@ -3742,64 +2504,16 @@ paymentApi.get_payment_tags(payment_id, api_key, api_secret)
 
 > Example Response:
 
-```shell
-# Subset of headers returned when specifying -v curl option
-< HTTP/1.1 200 OK
-<
-[
-  {
+```json
+{
     "tagId": "890e3b13-3114-478b-9365-50f1a2682143",
     "objectType": "PAYMENT",
     "objectId": "8fe697d4-2c25-482c-aa45-f6cd5a48186d",
     "tagDefinitionId": "353752dd-9041-4450-b782-a8bb03a923c8",
     "tagDefinitionName": "foo",
```

Comment:
Same comment as above, I think an empty body is sent, could you please confirm and update if required?